src/cloud-resources/src/crd/materialize.rs

```diff
@@ -43,39 +43,41 @@ pub mod v1alpha1 {
     #[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize, JsonSchema)]
     #[serde(rename_all = "camelCase")]
     pub struct MaterializeCertSpec {
-        // Additional DNS names the cert will be valid for.
+        /// Additional DNS names the certificate will be valid for.
         pub dns_names: Option<Vec<String>>,
-        // Duration the certificate will be requested for.
-        // Value must be in units accepted by Go time.ParseDuration
-        // https://golang.org/pkg/time/#ParseDuration.
+        /// Duration the certificate will be requested for.
+        /// Value must be in units accepted by Go `time.ParseDuration`
+        /// https://golang.org/pkg/time/#ParseDuration.
         pub duration: Option<String>,
-        // Duration before expiration the certificate will be renewed.
-        // Value must be in units accepted by Go time.ParseDuration
-        // https://golang.org/pkg/time/#ParseDuration.
+        /// Duration before expiration the certificate will be renewed.
+        /// Value must be in units accepted by Go `time.ParseDuration`
+        /// https://golang.org/pkg/time/#ParseDuration.
         pub renew_before: Option<String>,
-        // Reference to an Issuer or ClusterIssuer that will generate the certificate.
+        /// Reference to an `Issuer` or `ClusterIssuer` that will generate the certificate.
         pub issuer_ref: Option<CertificateIssuerRef>,
-        // Additional annotations and labels to include in the Certificate object.
+        /// Additional annotations and labels to include in the Certificate object.
         pub secret_template: Option<CertificateSecretTemplate>,
     }
     #[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize, JsonSchema)]
     pub enum MaterializeRolloutStrategy {
-        // Default. Create a new generation of pods, leaving the old generation around until the
-        // new ones are ready to take over.
-        // This minimizes downtime, and is what almost everyone should use.
+        /// Create a new generation of pods, leaving the old generation around until the
+        /// new ones are ready to take over.
+        /// This minimizes downtime, and is what almost everyone should use.
         #[default]
         WaitUntilReady,
 
-        // WARNING!!!
-        // THIS WILL CAUSE YOUR MATERIALIZE INSTANCE TO BE UNAVAILABLE FOR SOME TIME!!!
-        // WARNING!!!
-        //
-        // Tear down the old generation of pods and promote the new generation of pods immediately,
-        // without waiting for the new generation of pods to be ready.
-        //
-        // This strategy should ONLY be used by customers with physical hardware who do not have
-        // enough hardware for the WaitUntilReady strategy. If you think you want this, please
-        // consult with Materialize engineering to discuss your situation.
+        /// WARNING!!!
```

Comment:
In my pr, I tweaked the json file to use our {{< warning >}} and {{< /warning >}} instead of the the two "WARNING!!!".